pkg/skaffold/test/custom/custom.go

```diff
@@ -32,34 +36,42 @@ import (
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 )
 
-// for tests
-var doRunCustomCommand = runCustomCommand
+var (
+	// for tests
+	testContext = retrieveTestContext
+
+	// Image is an environment variable key, whose value is the
+	// fully qualified image name passed in to a custom command.
+	Image = "IMAGE"
```

Comment:
Can't find any reference for `Image`

Reply:
The environment variable `Image` is just set by the custom test to make the image name accessible to the custom command at run time.

Reply:
but it's not being used anywhere. In `getEnv` function you're using literal `"IMAGE_NAME"` again.

Reply:
Yes. Removed it. Thanks.